pmd-core/src/main/java/net/sourceforge/pmd/RuleSetFactory.java

```diff
@@ -61,9 +52,6 @@
     private static final String DESCRIPTION = "description";
     private static final String UNEXPECTED_ELEMENT = "Unexpected element <";
     private static final String PRIORITY = "priority";
```

Comment:
Is this constant still needed in this class (for parsing rule sets)? I think, it's a rule-specific property...

Reply:
It's still used in `parseRuleSetReferenceNode` apparently. On ruleset references, the mentioned priority overrides the priority of each rule, see lines 480 and 497